### ISSUE TITLE
Manage limesurvey config.php

### ISSUE THREAD

**Issue by jacksgt** [vision-it/vision-limesurvey#8]
Currently this module does not manage limesurvey configuration `config.php`.
While the container image `martialblog/limesurvey` does an initial bootstrap of the file, it won't change its content once it has been created.
This leads to a lot of consistency issues.

For example, once the password has been set, it cannot be changed via Puppet later on.

**Comment by martialblog** [comment]
If think there was an issue when the config.php exists the application with initialize the first time...

**Comment by jacksgt** [comment]
Yes, also because the container image only initializes the config from env variables if there is no config file present.

```
if file 'config.php' exists:
    manage content with file_line (e.g. ensure "'password' => 'foo'," is set)
```

**Comment by martialblog** [comment]
Maybe we can use something like that:

```
$file = "config.php"

  exec { "chk_${file}_exist":
    command => "true",
    path    =>  ["/usr/bin","/usr/sbin", "/bin"],
    onlyif  => "test -f ${file}"
  }

  file {"config.php":
    ensure => file,
   content => template('vision_limesurvey/config.php.erb'),
    require => Exec["chk_${file}_exist"],
  }
```

Gonna check it out next week.

**Comment by martialblog** [comment]
This should do the trick: https://github.com/vision-it/vision-limesurvey/tree/manage-config

TODO: Gotta update the config to support sockets
